lbrynet/lbrynet_daemon/LBRYDaemonServer.py

```diff
@@ -67,6 +67,7 @@ def requestReceived(self, command, path, version):
         from twisted.web.http import parse_qs
         if self.do_log:
             print '%f Request Received' % time.time()
+            print self.content
```

Comment:
remove this.  Its garbage.  I removed it in an earlier commit - was there not a conflict?